Firebase/android/app/src/main/java/com/novoda/bonfire/view/BubblyLinearLayout.java

```diff
@@ -0,0 +1,78 @@
+package com.novoda.bonfire.view;
+
+import android.content.Context;
+import android.content.res.TypedArray;
+import android.graphics.Canvas;
+import android.graphics.Paint;
+import android.graphics.RectF;
+import android.util.AttributeSet;
+import android.widget.LinearLayout;
+
+import com.novoda.bonfire.R;
+
+public class BubblyLinearLayout extends LinearLayout {
```

Comment:
Shouldn't be a `LinearLayout` but rather a `FrameLayout` -- see https://github.com/novoda/spikes/pull/107/files#r67866404